clients/admin-ui/src/features/data-discovery-and-detection/utils/getResourceName.ts

```diff
@@ -0,0 +1,19 @@
+import { DiscoveryMonitorItem } from "~/features/data-discovery-and-detection/types/DiscoveryMonitorItem";
+
+const TOP_LEVEL_FIELD_URN_PARTS = 5;
+
+const getResourceName = (resource: DiscoveryMonitorItem) => {
+  const splitUrn = resource.urn.split(".");
```

Comment:
we had an issue we fixed in datasets where Mozilla had a "." in one of their field names which broke the URN. Is that not a concern with the D&D stuff? Like can we use a hex code that looks like a "." instead or something?